packages/main/src/DayPicker.js

```diff
@@ -441,6 +486,70 @@ class DayPicker extends UI5Element {
 		}
 	}
 
+	_navToStartEndDayOfTheMonth(event, start) {
+		event.preventDefault();
+
+		const currentItem = this._itemNav._getCurrentItem();
+		let currentTimestamp = parseInt(currentItem.getAttribute("data-sap-timestamp")) * 1000;
+		let oCalDate = CalendarDate.fromTimestamp(currentTimestamp, this._primaryCalendarType);
+
+		if (currentItem.classList.contains("ui5-dp-item--othermonth")) {
+			return;
+		}
+
+		if (this._isOutOfSelectableRange(oCalDate.toLocalJSDate())) {
+			return;
+		}
+
+		oCalDate.setDate(1)
+
+		if (!start) {
+			// set the day to be the last day of the current month
+			oCalDate.setMonth(oCalDate.getMonth() + 1, 0);
+		}
+		
+		currentTimestamp = oCalDate.valueOf() / 1000;
+		let newItem = this._itemNav._getItems().find( item => item.timestamp == currentTimestamp );
+
+		this._itemNav.currentIndex = newItem._index;
+		this._itemNav.focusCurrent();
+	}
+
+	_incrementYears(event, forward, step) {
```

Comment:
I think that the name of this method should not indicate whether it is going to increment or decrement the years, because it depends on the arguments. It should be called something neutral like changeYears. Moreover it would be good to add docs explaining the arguments. Lastly it would be good if you set default values for the ```forward``` and ```step``` arguments. This way, you would not need to always pass values when invoking the method

Reply:
Documentation is added and method name is changed. I've tested with default arguments and didn't like the result for this one, as this method is used with different sets of arguments already.